doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -159,9 +159,11 @@ and ``coqtop``, unless stated otherwise:
   short or partially qualified names.
 
   .. seealso:: Section :ref:`names-of-libraries`.
-:-top *dirpath*: Set the toplevel module name to :n:`@dirpath` instead of ``Top``.
-  Not valid for `coqc` as the toplevel module name is inferred from the
-  name of the output file.
+:-top *dirpath*: Set the logical module name to :n:`@dirpath` for a
+  `coqtop` interactive session. If no module name is specified,
+  `coqtop` will default to ``Top``. This option is not accepted by
+  `coqc` as the logical module name is inferred there from the name of
+  the input file and corresponding `-R` / `-Q` options.
```

Comment:
```suggestion
:-top *dirpath*: Set the toplevel module name to :n:`@dirpath` for
  `coqtop`. If no module name is specified,
  `coqtop` defaults to ``Top``. `coqc` does not accept this option
  because the toplevel module name is inferred from the name of
  the input file and the `-R` / `-Q` options.
```

I'd stick with "toplevel module" to correspond to the option `-top`.  (Also, module names are always logical names, not physical names, so "logical" doesn't add anything.)

I assume coqc doesn't support providing `-R`/`-Q` independently for each file being compiled.